### PR TITLE
chore: Update to Meteor 1.11.1

### PR DESCRIPTION
**Pull request by jstarpl**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

This PR updates the Meteor framework to a security release 1.11.1

* **What is the new behavior (if this is a feature change)?**

[Meteor 1.11.1 Release notes](https://docs.meteor.com/changelog.html#v111120200916)

* **Other information**:

**Status**

- [x] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

**Comment by jstarpl** [comment]
[tv-automation-meteor-base](https://github.com/nrkno/tv-automation-meteor-base) seems to have already been updated.

**Comment by Julusian** [comment]
@jstarpl you should update the image versions used in https://github.com/nrkno/tv-automation-server-core/blob/release26/.circleci/config.yml to use that 1.11.1 meteor-base image

**Comment by jstarpl** [comment]
Good find @Julusian !

File: meteor/.meteor/release [hunk 1]
```diff
@@ -1,2 +1 @@
-METEOR@1.11
-# Note: Remember to update the base docker image in .circleci/config.yml when updating this!
```
Comment (nytamin) [inline meteor/.meteor/release:2]: Could we keep this comment? I think it'll help future updaters.
(I know, it was probably removed when doing `meteor update`, but still)

**Review by nytamin — APPROVED** [review]

**Review by sparkpunkd — APPROVED** [review]
Looks good. Runs up OK on my local machine.

**Pull request merged**